src/spec.ts

```diff
@@ -591,13 +590,12 @@ function normalizePathOverlay(spec: NormalizedUnitSpec, config: Config = {}): No
   }
 
   if (lineOverlay) {
-    const {interpolate} = markDef;
     layer.push({
       ...(projection ? {projection} : {}),
       mark: {
+        ...pick(markDef, ['clip', 'interpolate']),
```

Comment:
Not sure why you have to change the order here. I think putting mark type at the top is more desirable.

Reply:
Only if we never copy the whole properties object.

Reply:
Changed the order and rebased.